cross/build-rootfs.sh

```diff
@@ -38,6 +40,14 @@ for i in "$@"
         __UbuntuPackages="build-essential libunwind8-dev gettext symlinks liblttng-ust-dev libicu-dev"
         __MachineTriple=aarch64-linux-gnu
         ;;
+        arm-softfp)
+        __BuildArch=arm-softfp
+        __UbuntuArch=armel
+        __UbuntuRepo="http://ports.ubuntu.com/"
+        __UbuntuPackages="build-essential lldb-3.6-dev libunwind8-dev gettext symlinks liblttng-ust-dev libicu-dev"
+        __MachineTriple=arm-linux-gnueabi
+        __UbuntuCodeName=precise
```

Comment:
Not sure of the solution to this, but 12.04 is EOL next year. Might be worth thinking of a plan for future support.